### PR TITLE
Sketch bin

### PR DESCRIPTION
**Pull request by tmcw**
This starts to experiment with a binary interface for mapbox-sdk-js cli usage

### CODE REVIEW

**Comment by sbma44** [comment]
over at the python SDK we wound up going with `mbx`: https://github.com/mapbox/mapbox-sdk-py/issues/10

not sure whether or not there's a payoff to standardizing the CLI name across languages. could make scripts work more often, could make debugging more confusing for unsophisticated users.

**Comment by tmcw** [comment]
It seems like having multiple binaries called `mbx` would make things really confusing given that they'd shadow each other in PATH

**Comment by yhahn** [comment]
Maybe we need a cross-language package manager to arbitrate called `mpm`

**Comment by sbma44** [comment]
@tmcw yeah, agreed. I should also probably note that since I wrote that comment, efforts have been made to decouple the python CLI tool and the python SDK. I'm decreasingly sure that there's a good case for bringing these kinds of patterns across languages. (emulating invocation styles/param names seems legit, to the extent it's practicable).

also obvs I strongly endorse @yhahn's suggestion

**Comment by tmcw** [comment]
:+1: Also per discussion with @perrygeo I'm actually leaning toward mapbox-sdk-py being the 'official' CLI wrapper, since Python is installed by default on more systems and would be easier to get running with. I _personally_ really like having a CLI for sdk-js basically as a way to test stuff (this PR for instance was just so I could figure out #59), so I'm kind of into releasing this quietly and having it not be the user-recommended thing, but have the Python version be that.

**Comment by yhahn** [comment]
`mbxjs`

and or frosted

**Comment by perrygeo** [comment]
As mentioned by @sbma44 - we're in the process of moving the CLI over to a separate repo (https://github.com/mapbox/mbx-cli) in order to avoid tying the Python SDK too tightly to an official CLI. This allows

* the READMEs, docs, and branding to be adjusted for the appropriate audience
* the implementation or language behind the CLI can change in the future without disrupting user's scripts

On a related note, we're in the early stages of the CLI design so we're hoping to get some input before any initial release. @sgillies will be starting work in earnest next week so we'll track any design questions in the `mbx-cli` tickets.

File: package.json [hunk 1]
```diff
@@ -3,6 +3,9 @@
   "version": "0.10.0",
   "description": "interface to mapbox services",
   "main": "lib/mapbox.js",
+  "bin": {
+    "mapbox-sdk": "./bin/mapbox-sdk"
```
Comment (zeke) [inline package.json]: Why not just `mapbox`? Is that name used by some other tool?

Comment (yhahn) [inline package.json]: :trollface:

Comment (zeke) [inline package.json]: Not trolling!

Comment (yhahn) [inline package.json]: I know :smile: We actually have an internal command we've been working on called `mapbox` and we're trolling ourselves by not just swapping names (public: mapbox, internal: mbx or something else).

Comment (zeke) [inline package.json]: > public: mapbox, internal: mbx

I seeee.

**Comment by karenzshea** [comment]
`mapbox-cli` namespace is all urs @tmcw

**Comment by tmcw** [comment]
mapbox-cleee

**Comment by tmcw** [comment]
Closing: let's dedicate efforts to https://github.com/mapbox/mapbox-cli-py !

**Pull request closed without merging**